### DISCUSSION TITLE
Is there's any possible way to show both positive and negative numbers in vis-timeline time axis instead of defualt date given by the Library

### DISCUSSION THREAD

**Discussion by devin1996** [visjs/vis-timeline#1698] [Q&A]
Is there's any possible way to show both positive and negative numbers in vis-timeline instead of default date given by the Library

I was trying to implement a timeline which starts from a negative number (-20) and ends with a Positive number (20). By Default vis-time line time axis only supports for Date Formats, in both major and minor labels in time axis for vis-timeline

I am trying to implement this timeline using Angular

Here are the current configurations given to timeline

```
// time line options
  getOptions() {
    this.options = {
      start: moment().subtract(10, 'm').toISOString(),
      end: moment().add(40, 'm').toISOString(),
      max: new Date(2023, 10, 21),
      min: new Date(2023, 8, 20),
      timeAxis: { scale: 'minute', step: 1 },
      zoomMax: 900000,
      zoomMin: 60000, // 1 min
    };
  }
```
 

A possible way to show both positive and negative numbers in vis-timeline instead of default date given by the Library

**Comment by sudarakas** [comment]
We can overlay the visjs-timeline's major label section with a separate custom label container. 

Try this example code snippet 
[VISJS Timeline Custom Labels](https://codepen.io/sudarakas/pen/ZEgmqma)
